### PR TITLE
Fixes 1 - 524 tests pass 38 run tests to go!

### PR DESCRIPTION
**Pull request by albertjan**
- LSuffix / Silent octal in tokenizer
- typo's in ast.js
- Sk.ast. missing from compile.js
- port `ast_for_flow_stmt`
- fix multiline tripple quoted strings
- fix multiline single quoted strings

`run: 524/562 (+1 disabled)`

### CODE REVIEW

**Comment by albertjan** [comment]
I'm on the hunt to fix case 278

**Comment by albertjan** [comment]
278 passes 💃

**Comment by albertjan** [comment]
I will attempt `ast_for_call` next ... it's a big'un

**Comment by meredydd** [comment]
This. Is. Awesome.

File: src/compile.js [hunk 1]
```diff
@@ -1243,6 +1243,9 @@ Compiler.prototype.cfor = function (s) {
 };
 
 Compiler.prototype.craise = function (s) {
+    // @meredydd compile.c doesn't have this function anymore. raise doesn't have a type anymore
```
Comment (albertjan) [inline src/compile.js]: I left this one for you @meredydd 😁

**Comment by albertjan** [comment]
Officially super seeded by £9

**Comment by meredydd** [comment]
Closing (superseded)

**Pull request closed without merging**